src/FileDialog.cpp

```diff
@@ -70,3 +70,8 @@ void FileDialog::setFileDialogPath(const QString& new_path)
         break;  // Do nothing
     }
 }
+
+QString FileDialog::getSqlDatabaseFileFilter()
+{
+    return Settings::getValue("General", "DBFileExtensions").toString();
```

Comment:
I would suggest always adding ```+ ";;" + QObject::tr("All files (*)")``` here and remove it from the list of editable file extensions. It doesn't seem to make a lot of sense to let the user edit or delete this option and it's probably better to always have it as a automatically added fallback here.